lua/snacks/lazygit.lua

```diff
@@ -77,7 +77,7 @@ local function env(opts)
 
     if vim.v.shell_error == 0 and #lines > 1 then
       config_dir = vim.split(lines[1], "\n", { plain = true })[1]
-      vim.env.LG_CONFIG_FILE = vim.fs.normalize(config_dir .. "/config.yml" .. "," .. opts.theme_path)
+      vim.env.LG_CONFIG_FILE = (vim.env.LG_CONFIG_FILE or vim.fs.normalize(config_dir .. "/config.yml")) .. "," .. opts.theme_path
```

Comment:
This will keep extending that env var, so won't work as expected.
Add a module level `LG_CONFIG_FILE = vim.env.LG_CONFIG_FILE` once and use that instead so we don't keep extending it with the theme path every time lazygit is laucnhed

Reply:
Moved env lookup to outside of `env` function. Resolves repeated appending of theme config